src/XRP/wallet.ts

```diff
@@ -27,8 +22,13 @@ export interface WalletGenerationResult {
  * A wallet object that has an address and keypair.
  */
 class Wallet {
-  private readonly publicKey: string
-  private readonly privateKey: string
+  /**
+   * The default derivation path to use with BIP44.
+   */
+  public static defaultDerivationPath = "m/44'/144'/0'/0/0"
+
+  public readonly publicKey: string
+  public readonly privateKey: string
```

Comment:
Make these two properties public so I can get rid of their `getter()` functions.

They are marked `readonly`, so they are effectively `private` with `getter()` functions.